tests/snmp/test_snmp.py

```diff
@@ -0,0 +1,5 @@
+from common.utilities import wait_until
```

Comment:
here is a better way.

define a conftest.py in this tests/snmp/ directory. 

define your fixture in this conftest.py.

check example below.

https://docs.pytest.org/en/latest/writing_plugins.html#conftest-py-local-per-directory-plugins

Reply:
@lguohan , thanks. I define a conftest.py in tests/snmp/ directory to wait for snmp service to be started. Could you please review it?